### PR TITLE
Refactor subscribe JS

### PR DESCRIPTION
**Pull request by Rotzbua**
## Change

Remove jQuery dependency. Preparation for #203.

### CODE REVIEW

**Comment by keunes** [comment]
Hi @Rotzbua,
Just FYI: I think this needs a proper code review and I can't do that. We'll have to wait until ByteHamster is available again, starting from next week.

File: deeplink/subscribe/index.html [hunk 1]
```diff
@@ -24,20 +24,25 @@
...
     return false;
   };
 
-  jQuery("#urlTextBox").text(getUrlParameter("url"));
-  jQuery("#retryButton").click(function() {
-    window.open("antennapod-subscribe://" + getUrlParameter("url"));
-  });
+  const urlTextBox = document.getElementById("urlTextBox");
+  const retryButton = document.getElementById("retryButton");
+  if(getUrlParameter("url") === false) {
```
Comment (ByteHamster) [inline deeplink/subscribe/index.html:37]:
```suggestion
  if (getUrlParameter("url") === false) {
```

File: deeplink/subscribe/index.html [hunk 2]
```diff
@@ -24,20 +24,25 @@
...
-    var sPageURL = window.location.search.substring(1),
-      sURLVariables = sPageURL.split('&'), sParameterName, i;
-    for (i = 0; i < sURLVariables.length; i++) {
-      sParameterName = sURLVariables[i].split('=');
-      if (sParameterName[0] === sParam) {
-        return typeof sParameterName[1] === undefined ? true : decodeURIComponent(sParameterName[1]);
-      }
+  const getUrlParameter = (sParam) => {
+    const paramValue = new URLSearchParams(window.location.search.substring(1)).get(sParam) ?? '';
+    if(0 < paramValue.length) {
```
Comment (ByteHamster) [inline deeplink/subscribe/index.html:29]:
```suggestion
    if (0 < paramValue.length) {
```

**Review by ByteHamster — COMMENTED** [review]
Looks good to me. I would make the whitespace consistent, as suggested in my comments.

**Pull request merged**